src/2_RTB/VerbaliInterni/VerbaleInterno_241123/content.typ

```diff
@@ -0,0 +1,45 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+
+== Assegnazione dei ruoli
+Su proposta dei componenti è stata assegnata questa  distribuzione dei ruoli:
+- Responsabile: Giacomo D'Ovidio;
+- Amministratore: Riccardo Costantin;
+- Analista: Riccardo Toniolo;
+- Verificatore: Nancy Kalaj;
+- Programmatore: Matteo Rango;
+- Programmatore: Simone Caregnato;
+Il ruolo di Progettista non è stato assegnato, in quanto per il momento ha operatività limitata.
+La presenza di due programmatori è dovuta al fatto che il Team in accordo con la proponente vuole sviluppare una prima versione del PoC entro la fine dello sprint corrente. 
+
+== Cambio di ruoli settimanale
+Per favorire una rotazione più rapida e un carico di lavoro più omegeneo il team ha deciso di introdurre la possibilità di poter cambiare i ruoli una volta raggiunta la metà temporale di ogni sprint. Nell'ottica dello sprint iniziato in data odierna, il cambio dei ruoli avverrà ipoteticamente venerdi 1 Dicembre.
+Si precisa che il cambio dei ruoli è opzionale e non necessariamente riguarda la totalità dei ruoli e dei membri.
```

Comment:
- "Per favorire una rotazione più rapida e un carico di lavoro distribuito in modo più omegeneo il team ha deciso di introdurre la possibilità di cambiare i ruoli una volta raggiunta la metà temporale di ogni sprint."
- venerdi --> venerdi'
- dopo l'ultima frase aggiungerei il criterio secondo cui cambiamo i ruoli, ossia il fatto che e' legato al completamento e verifica delle issues assegnate a un determinato ruolo: se qualcuno ha esaurito le proprie issues e' libero di assumersi un altro ruolo, altrimenti mantiene lo stesso ruolo e continua a lavorare alle issues non terminate.